### DISCUSSION TITLE
Add executing multiple commands at once to CustomText

### DISCUSSION THREAD

**Discussion by danyadanya2005** [Zrips/CMI#7838] [Feature Requests]
Hello, I've run into this problem. i need to execute two commands at once, but the last command with titlemsg is being executed. first mask command fails
CustomText
<T>&e&lCLICK</T><CCI>disguiseplayer [playerName] Cat</CCI><CCI>cmi titlemsg [playerName] &a&lYou successfully disguised yourself -in:20 -out:20 -keep:30</CCI>
Please add executing multiple commands at once to CustomText?
example
<T>&e&lCLICK</T><CCI>disguiseplayer [playerName] Cat; cmi titlemsg [playerName] &a&lYou successfully disguised yourself -in:20 -out:20 -keep:30</CCI>

**Comment by CloudeLecaw** [comment]
may a seperator ;; like in attached command to allow mupliple command would be nice.

at least i like your idea
hands  up🤭

**Comment by MrCrazys** [comment]
> Hello, I've run into this problem. i need to execute two commands at once, but the last command with titlemsg is being executed. first mask command fails CustomText &e&lCLICKdisguiseplayer [playerName] Catcmi titlemsg [playerName] &a&lYou successfully disguised yourself -in:20 -out:20 -keep:30 Please add executing multiple commands at once to CustomText? example &e&lCLICKdisguiseplayer [playerName] Cat; cmi titlemsg [playerName] &a&lYou successfully disguised yourself -in:20 -out:20 -keep:30

As far as I know, this is a Minecraft limitation, so it can't really be fixed.

**Comment by danyadanya2005** [comment]
this is implemented in the seversign plugin. Why can't this be implemented in CMI? seversign commands are written on the plate

**Reply by MrCrazys** [reply]
> this is implemented in the seversign plugin. Why can't this be implemented in CMI? seversign commands are written on the plate

That's for signs. It's a different story for clicking on chat messages.

**Reply by CloudeLecaw** [reply]
> this is implemented in the seversign plugin. Why can't this be implemented in CMI? seversign commands are written on the plate

like allready ridden you cant count what works fir signs for chat.  with cmi you also can attach interactive commands to signs and blocks where you can use multiple commands.

but this does not mean its possible for chat.

**Comment by danyadanya2005** [comment]
it is sad
